engine/execution/computation/computer/computer.go

```diff
@@ -241,9 +241,22 @@ func (e *blockComputer) executeSystemCollection(
 
 	err = e.executeTransaction(tx, colSpan, collectionView, programs, systemChunkCtx, collectionIndex, txIndex, res)
 	txIndex++
+
 	if err != nil {
+		return txIndex, err
+	} else if res.TransactionResults[txIndex].ErrorMessage != ""{
+		// This log is used as the data source for an alert on grafana.
+		// The system_chunk_error field must not be changed without adding the corresponding
+		// changes in grafana. https://github.com/dapperlabs/flow-internal/issues/1546
+		e.log.Err(fmt.Errorf(res.TransactionResults[txIndex].ErrorMessage)).
```

Comment:
We use `Errorf` with an input which comes from external source. This is potentially exploitable and simply unnecessary.
No need to use `Err` only to use `fmt.Errorf` to wrap a string.